buildshiprun/handler.go

```diff
@@ -365,3 +378,11 @@ func getRegistryAuthSecret() string {
 	}
 	return ""
 }
+
+// BuildResult represents a successful Docker build and
+// push operation to a remote registry
+type BuildResult struct {
```

Comment:
Do you think we need to extract this struct in sdk?

Reply:
Sure